addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -218,6 +218,8 @@ def _get_mo_summary(self, production, components, operations, current_mo_cost, c
     def _get_unit_cost(self, move):
         if not move:
             return 0.0
+        if move.state == 'done':
+            return move.price_unit
```

Comment:
Not saying it's wrong, but how is that related to the commit/PR? :thinking:

Reply:
In case the move is "done" the value was taken on layers here https://github.com/odoo-dev/odoo/blob/saas-18.3-forecast-widget-fix-whe/addons/mrp_account/report/mrp_report_mo_overview.py. 
I just forget to specify this condition to non storable products